src/main/java/com/matera/account/account/AccountRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.matera.account.account;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AccountRepository extends JpaRepository<Account, Integer> {
+
```

Comment:
Here you can create methods that spring data recognizes by convention, like:

`public List<Account> findAllByClientId(UUID clientId);`